synapse/storage/databases/main/media_repository.py

```diff
@@ -327,7 +327,7 @@ async def get_local_media_ids(
         if include_protected_media is False:
             # Do not include media that has been protected from quarantine
             sql += """
-                AND safe_from_quarantine = 0
+                AND safe_from_quarantine = false
```

Comment:
Whoops. Thanks for catching! Fixed in https://github.com/matrix-org/synapse/pull/12977.

Reply:
Do we have a minimum supported sqlite version? Should CI have caught this?

Reply:
Our [deprecation policy](https://matrix-org.github.io/synapse/latest/deprecation_policy.html#deprecation-policy-for-platform-dependencies) only mentions Python and PostgreSQL. Our unit tests look to only test different postgres versions as well.

I'm not sure if we do have a minimum supported version of SQLite, but https://matrix-org.github.io/synapse/latest/development/database_schema.html?highlight=database#boolean-columns does mention older versions, and it may be good to test an older SQLite in CI (maybe similar mistakes have slipped in already and we don't even know!).

Reply:
I thought at least one of our CI runs did old SQLite by using an old version of ubuntu? Though it does look like our CI is using latest ubuntu images everywhere...

Reply:
let's take this to https://github.com/matrix-org/synapse/issues/12983